docs/sphinx/install.rst

```diff
@@ -428,9 +425,145 @@ when connecting via SSH, which will launch the CUDA Quantum container and connec
+and configure it with 
+
+.. code-block:: console
+
+    ngc config set
+
+entering the API key you just generated when prompted.
+
+.. note::
+
+  The rest of this section assumes you have CLI version 3.33.0. If you 
+  have an older version installed, you can upgrade to the latest version using the command
+
+  .. code-block:: console
+
+      ngc version upgrade 3.33.0
+  
+  See also the `NGC CLI documentation <https://docs.ngc.nvidia.com/cli/index.html>`__
+  for more information about available commands.
+
+You can see all information about available compute resources and ace instances
+with the command 
+
+.. code-block:: console
+
+    ngc base-command ace list
+
+Confirm that you can submit a job with the command
+
+.. code-block:: console
+
+    ngc base-command job run \
+      --name Job-001 --total-runtime 60s \
+      --image nvcr.io/nvidia/nightly/cuda-quantum:latest --result /results \
+      --ace <ace_name> --instance <instance_name> \
+      --commandline 'echo "Hello from DGX Cloud!"'
+
+replacing `<ace_name>` and `<instance_name>` with the name of the ace and instance you want 
+to execute the job on.
+You should now see that job listed when you run the command
+
+.. code-block:: console
+
+    ngc base-command job list
+
+Once it has completed you can download the job results using the command
+
+.. code-block:: console
+
+    ngc base-command result download <job_id>
+
+replacing `<job_id>` with the id of the job you just submitted.
+You should see a new folder named `<job_id>` with the job log that contains 
+the output "Hello from DGX Cloud!".
+
+For more information about how to use the NGC CLI to interact with DGX Cloud, 
+we refer to the `NGC CLI documentation <https://docs.ngc.nvidia.com/cli/index.html>`__.
+
+Use JupyterLab
++++++++++++++++++++++++++++++++
+
+Once you can :ref:`run jobs on DGX Cloud <dgx-cloud-setup>`, you can launch an interactive job 
+to use CUDA Quantum with `JupyterLab <https://jupyterlab.readthedocs.io/en/latest/>`__ 
+running on DGX Cloud:
+
+.. code-block:: console
+
+    ngc base-command job run \
+      --name Job-interactive-001 --total-runtime 600s \
+      --image nvcr.io/nvidia/nightly/cuda-quantum:latest --result /results \
+      --ace <ace_name> --instance <instance_name> \
+      --port 8888 --commandline 'jupyter-lab-setup <my-custom-token> --port=8888'
+
+Replace `<my-custom-token>` in the command above with your own password that you can freely choose.
+You will use this password to authenticate with JupyterLab;
+Go to the `job portal <https://bc.ngc.nvidia.com/jobs>`__, click on the job you just launched, and click on the link
+under |:spellcheck-disable:|"URL/Hostname"|:spellcheck-enable:| in Service Mapped Ports. 
+
+.. note::
+
+  It may take a couple of minutes for DGX Cloud to launch and for the URL to become active;
+  if you encounter a "404: Not Found" error, be patient and try again in a couple of minutes.
```

Comment:
Despite your comment, I was impatient and assumed it wasn't working, so I killed the job a few times before finally succeeding. The clarification below might help encourage people to wait longer, but feel free to ignore if you feel like you already have it covered.
```suggestion
  It may take a couple of minutes for DGX Cloud to launch and for the URL to become active, even after it appears in the Service Mapped Ports section;
  if you encounter a "404: Not Found" error, be patient and try again in a couple of minutes.
```

Reply:
I was hoping the *really* long wait time was an artifact of me having to connect to VPN across the world, but I guess not... That is rather suboptimal, but nothing we can do anything about at this time.